### DISCUSSION TITLE
Month and minute have the same abbreviation (m) in user's discussion history

### DISCUSSION THREAD

**Discussion by reregaga** [community/community#43404] [Discussions]
### Select Topic Area

Product Feedback

### Body

The month and the minute have the same abbreviation (`m`) if you look at [your discussion list](https://github.com/discussions), this can be confusing.  
I think we should call them something different, maybe `mo` and `mi`.  
Current example for responses 17 minutes ago and 2 months ago:  
>![image](https://user-images.githubusercontent.com/76629445/210852064-1e43e953-c35e-45e0-ba2e-87705d0b8d6c.png)

**Comment by bic-ed** [comment]
Same in the following section of the github home page.
